sequencing/ATACSeq/config/config.txt

```diff
@@ -23,16 +48,15 @@ export XCHRBED=/lustre/projects/Research_Project-MRC190311/references/GeneLists/
 export UCSCRef=/lustre/projects/Research_Project-MRC190311/references/UCSC/
 export GENOMESIZE=/lustre/projects/Research_Project-MRC190311/references/grch38/genome.info
 export SEGREFGENOME=/lustre/projects/Research_Project-MRC190311/references/Segway/GRCh38/Sequence/WholeGenomeSegwayFasta/genome.char.fa
-
+export SNP=${GENODIR}/hg38/allchr_filt_rsq_maf.vcf.gz
```

Comment:
somewhere (README I think) we need to clarify what this file is, how it is created etc. Also how robust is the pipeline if it is not present?